test/testmachinery/shoots/logging/seed_logging_stack.go

```diff
@@ -152,8 +149,8 @@ var _ = ginkgo.Describe("Seed logging testing", func() {
 		)
 
 		ginkgo.By("Get Loki tenant IDs")
-		userID := getXScopeOrgID(grafanaUsersIngress.GetAnnotations())
-		operatorID := getXScopeOrgID(grafanaOperatorsIngress.GetAnnotations())
+		userID := "user" // TODO we have a single Grafana now, so the multi-tenancy feature in Loki shall be removed
```

Comment:
Can we assign a person for this TODO? Maybe

```suggestion
		userID := "user" // TODO(vlvasilev): we have a single Grafana now, so the multi-tenancy feature in Loki shall be removed
```

@vlvasilev , any objections?

Reply:
See https://github.com/gardener/gardener/pull/7007/commits/9c949e6d2ee656fe28a7d6ad1ff803c153bd879d#diff-bae2f5d169b1d2b6bb9d41bdef0985421af53d4a8b76dda58f098c96781972b6R152